packages/core/core-flows/src/fulfillment/workflows/create-fulfillment.ts

```diff
@@ -16,20 +10,6 @@ export const createFulfillmentWorkflow = createWorkflow(
   ): WorkflowData<FulfillmentDTO> => {
     const fulfillment = createFulfillmentStep(input)
 
-    const link = transform(
-      { order_id: input.order_id, fulfillment },
-      (data) => {
-        return [
-          {
-            [Modules.ORDER]: { order_id: data.order_id },
-            [Modules.FULFILLMENT]: { fulfillment_id: data.fulfillment.id },
-          },
-        ]
-      }
-    )
-
-    createLinkStep(link)
```

Comment:
suggestion: should we then remove the order_id from the input?

Reply:
Which input do you mean?

Reply:
the workflow input

Reply:
I removed it from `CreateFulfillmentWorkflowInput` but we need the order_id in the input to fetch the order.